crafters/nlp/TikaExtractor/__init__.py

```diff
@@ -0,0 +1,60 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import requests
+from requests.exceptions import ConnectionError
+from jina.executors.crafters import BaseCrafter
+
+TIKA_URL = 'http://0.0.0.0:9998'
```

Comment:
I guess this is the url to connect to the jvm underlying right? Isnt there some py4j connection with Tika?

Reply:
Its the connection to the REST api provided by Tika. Yes you are right, might be possible to directly connect to the jvm